src/main/webapp/app/tutor-exercise-dashboard/tutor-exercise-dashboard.component.ts

```diff
@@ -320,4 +329,12 @@ export class TutorExerciseDashboardComponent implements OnInit {
         // in the case of 'undefined' the complaint is not yet handled
         return 'The complaint still needs to be evaluated';
     }
+
+    calculateMoreFeedbackStatus(accepted?: boolean) {
+        if (accepted !== undefined) {
+            return 'More Feedback Request has already been evaluated';
```

Comment:
why do we not use translations here?

Reply:
Changed it to use translations for both complaint and more feedback request